plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/portforwading/PortForwardingWizardModel.java

```diff
@@ -73,6 +74,10 @@ public boolean getPortForwarding() {
 		return isPortForwarding(pod);
 	}
 
+	public boolean getPortForwardingAllowed() {
```

Comment:
Convention would be to make this a question such as 'isPortForwardingAllowed'.  Does that still work with databinding? I don't recall